app/views/static/_old_pricing_versions_list.html.erb

```diff
@@ -1,4 +1,4 @@
-<div><strong>Staršie verzie cenníka</strong></div>
+<div style="margin-top: 54px"><strong>Staršie verzie cenníka</strong></div>
```

Comment:
Hm, takto natvrdo?

Reply:
@jsuchal  Hej, ani mne sa to nepaci, `views/static` pouzivaju iny layout nez views https://ekosystem.slovensko.digital/sluzby/govbox/cennik , idu cez bootstrap 3.7 este a maju tam nejake staticke styly. Zmenil som to a mozeme to dat takto

![image](https://user-images.githubusercontent.com/44023416/69478833-c5cac800-0dee-11ea-9a45-82d1ca8958f5.png)
![image](https://user-images.githubusercontent.com/44023416/69478836-cfecc680-0dee-11ea-84a7-e95ae606d519.png)